ts-nextjs-typeorm/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Maybe also point to our tutorial?

Reply:
Yeah, this just the readme from the template. I haven't spent time on that for this sample.